keyboards/playkbtw/pk60/v1/rules.mk

```diff
@@ -0,0 +1,24 @@
+# MCU name
```

Comment:
Stuff that is common to both revisions can be placed in the top level rules.mk, instead of being duplicated here.